include/range/v3/view/any_view.hpp

```diff
@@ -545,15 +534,14 @@ namespace ranges
             CONCEPT_REQUIRES((Cat & category::sized) == category::sized)
             std::size_t size() const
```

Comment:
We still need this here, to override the size function from regular view_interface?

I think that's what got me confused, any_view never actually inherits any_view_impl, right?
So the inheritances are:
any_view -> view_facade -> view_interface
any_view.ptr_ -> any_input_view_interface [==input] 
any_view.ptr_ -> any_clonable_view_interface -> any_view_interface [>input]

And then the _impl types are the types that get stuffed into any_view.ptr_ to facilitate the delegation to the actual range's interface.

It all makes sense now💡